README.md

```diff
@@ -20,27 +20,29 @@ task queue.
    ```
 2. Create the `instance` directory.
    ```shell
-   mkdir instance 
+   mkdir instance
    ```
 3. Create a `config.py` file in the `instance` directory, based on the [`example.config.py`](config/example.config.py) file in the repository.
 4. Create a `settings.yaml` file in the `instance` directory, based on the [`example.settings.yaml`](config/example.settings.yaml) file in the repository.
    It is quite important to keep the progress bar setting disabled to not pollute the logs of the webapp.
 5. Start MongoDB and Redis.
-6. Run the Flask app (in production you should likely use [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/) 
+6. When running locally, populate MongoDB - tasks `dump` and `restore` in [fabfile.py](./fabfile.py)
```

Comment:
> If the Readme should be for everybody, there should be written another way of populating MongoDB, if you want to run the instance locally. Or we can paste the link to the dataset for download and then write how to import it. Or some steps to generate dataset from scratch, but it would take too much time.

I think the page README should be targetting anyone trying to run a local clone. There is currently no way to get a full database locally other than running the processing locally. We could add something that loads a full dataset archive into an empty database. However I currently do not have time for that. It would likely be a Flask command in the `admin/` module. Perhaps it could reuse some code from the update processing tasks just with the change that the dataset now doesn't come from local processing but just from some path on the filesystem. That way one could do a "manual" update of the web data with a run done on some other server/locally. Or perhaps a run that had some manually fixed data. That sounds like a nice feature!

> 
> I don't think that we should attach docs for the fabfile when it is internal and not useful for everybody

I was thinking of the fabfile as not really internal. However, I had another look and you are right, it makes a lot of assumptions about how the site is installed and run that it does not make sense to document it for an outsider.